x-pack/plugins/advanced_ui_actions/public/components/action_wizard/test_data.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiFieldText, EuiFormRow, EuiSelect, EuiSwitch } from '@elastic/eui';
+import { ActionFactory, ActionFactoryBaseConfig } from './action_wizard';
+
+export const dashboards = [
+  { id: 'dashboard1', title: 'Dashboard 1' },
+  { id: 'dashboard2', title: 'Dashboard 2' },
+];
+
+export const DashboardDrilldownActionFactory: ActionFactory<
+  {
+    dashboardId: string;
+    useCurrentDashboardFilters: boolean;
+    useCurrentDashboardDataRange: boolean;
+  },
+  {
+    dashboards: Array<{ id: string; title: string }>;
+  }
```

Comment:
Do you think this `Context` type could be internal to every `ActionFactory` instance and passed in via the constructor? 

e.g. instead of 

```
actionFactory.wizard({ context:: actionFactory.context, ... });
```

leave that out and since `wizard` fn is already internal to the `actionFactory`, it can use whatever it wants. 

and in here, instead of referencing `props.context.dashboards`, just use `dashboards` directly?  I don't think we have a use case for exposing this type/data.

Reply:
👍